### PR TITLE
aecpgenkill(): do in one pass instead of two

### PR DESCRIPTION
**Pull request by WalterBright**
Makes the logic easier to follow, too.

### CODE REVIEW

File: src/dmd/backend/gflow.d [hunk 1]
```diff
@@ -554,26 +554,33 @@ private void aecpgenkill(ref GlobalOptimizer go, int flowxx)
 {
     block* this_block;
 
-    /****************************
-     * Compute number of cp (copy propagation) elems.
-     * Mark cp elems by setting NFLaecp flag.
-     * Returns:
-     *      number of cp elems
+    /********************************
```
Comment (UplinkCoder) [inline src/dmd/backend/gflow.d:557]: please reinstate a better comment.
it should explain what `cp` means.

Comment (WalterBright) [reply src/dmd/backend/gflow.d:557]: cp is used as an abbreviation for "Copy Propagation" throughout the optimizer, and is defined elsewhere in gflow.d.

Comment (UplinkCoder) [reply src/dmd/backend/gflow.d:557]: well ¯\_(ツ)_/¯

**Comment by WalterBright** [comment]
```
[1]+  Stopped                 DMD=ldc ./semaphoreci.sh testsuite
```
useless error message in the semaphoreci log.

**Review by atilaneves — APPROVED** [review]

**Pull request merged**